.github/workflows/publish-c-apidocs.yml

```diff
@@ -15,16 +15,35 @@ jobs:
           sudo apt-get install libclang-cpp9
           wget https://www.doxygen.nl/files/doxygen-1.9.2.linux.bin.tar.gz
           tar xvzf doxygen-1.9.2.linux.bin.tar.gz
+      - name: Set vars
+        id: vars
+        run: echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
+      - name: Check outputs
+        run: echo ${{ steps.vars.outputs.sha_short }}
       - name: Run doxygen
         run: |
           mkdir -p build/doxygen
           cd docs/c_cxx
           ../../doxygen-1.9.2/bin/doxygen
-      - name: Publish
-        uses: JamesIves/github-pages-deploy-action@releases/v3
+      - name: Clean up
+        run:
+          rm -rf doxygen-1.9.2
+          rm doxygen-1.9.2.linux.bin.tar.gz
+      - uses: actions/checkout@v2
```

Comment:
Why do we make the effort to cleanup the doxygen stuff but not the libclang packages?

Reply:
It's not strictly necessary, but when I first ran it, I did not filter the checked in files and this binary was checked in. Do you think I should remove it?

Reply:
A comment saying we delete these so they don't get checked in would be fine.

Reply:
I removed them, as again this was an artifact of developing the flow